.build/build-checkstyle.xml

```diff
@@ -41,7 +41,7 @@
                     failOnViolation="true">
             <formatter type="plain"/>
             <formatter type="xml" tofile="${checkstyle.report.file}"/>
-            <fileset dir="${build.src.java}" includes="**/*.java"/>
+            <fileset dir="${build.src.java}" includes="**/*.java" excludes="build/**/*.java" />
```

Comment:
As far as I understand there shouldn't be the `build` directory under the `src`. Why do we need this change?

Reply:
@HadesArchitect ? 

I will (re-) check this…

Reply:
removing it.

Reply:
pushed, check new commits.